platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/views/CustomReconnectRoot.java

```diff
@@ -51,7 +52,7 @@ public interface CustomReconnectRoot<T, L> extends MerkleNode {
      *
      * @return a view representing this subtree
      */
-    LearnerTreeView<L> buildLearnerView(final ReconnectConfig reconnectConfig);
+    LearnerTreeView<L> buildLearnerView(final ReconnectConfig reconnectConfig, final ReconnectMapStats mapStats);
```

Comment:
missing nullity annotation for new param

Reply:
added